lms/static/scripts/frontend_apps/utils/url.ts

```diff
@@ -47,3 +47,43 @@ export function recordToSearchParams(
 
   return queryParams;
 }
+
+/**
+ * Converts a record into a query string.
+ * The result is prefixed with a question mark (`?`) if it's not empty.
+ *
+ * Examples:
+ *  * {} -> ''
+ *  * { foo: [] } -> ''
+ *  * { foo: 'bar' } -> '?foo=bar'
+ *  * { foo: 'bar', something: ['hello', 'world'] } -> '?foo=bar&something=hello&something=world'
```

Comment:
```suggestion
 * Examples:
 *    {} -> ''
 *    { foo: [] } -> ''
 *    { foo: 'bar' } -> '?foo=bar'
 *    { foo: 'bar', something: ['hello', 'world'] } -> '?foo=bar&something=hello&something=world'
```

Easier to read without the double `*`.

Reply:
It was intended to be a bulleted list 😅